src/main/java/Events.java

```diff
@@ -0,0 +1,15 @@
+public class Events extends Task {
+    String time;
+
+    public Events(boolean isDone, String eventName, String time) {
+
```

Comment:
I see an empty line here, but not in your constructors for your other classes. Perhaps you could remove this empty line to make it consistent with the rest?